cmd/deploy/deploy.go

```diff
@@ -92,15 +96,47 @@ func main() {
 	if !common.IsHexAddress(*sequencerAddressString) && len(*sequencerAddressString) > 0 {
 		panic("specified sequencer address is invalid")
 	}
+	sequencerAddress := common.HexToAddress(*sequencerAddressString)
+
 	if !common.IsHexAddress(*ownerAddressString) {
 		panic("please specify a valid rollup owner address")
 	}
+	ownerAddress := common.HexToAddress(*ownerAddressString)
+
 	if *prod && !common.IsHexAddress(*loserEscrowAddressString) {
 		panic("please specify a valid loser escrow address")
 	}
 
-	sequencerAddress := common.HexToAddress(*sequencerAddressString)
-	ownerAddress := common.HexToAddress(*ownerAddressString)
+	var batchPosters []common.Address
+	if len(*batchPostersString) > 0 {
+		batchPostersArr := strings.Split(*batchPostersString, ",")
+		for _, address := range batchPostersArr {
+			if !common.IsHexAddress(address) {
+				log.Error("invalid address in batch posters array", "address", address)
+				continue
+			}
+			batchPosters = append(batchPosters, common.HexToAddress(address))
+		}
+		if len(batchPosters) != len(batchPostersArr) {
+			panic("found at least one invalid address in batch posters array")
+		}
+	}
+	if len(batchPosters) == 0 {
+		log.Info("batch posters array was empty or invalid, defaulting to sequencer address")
```

Comment:
the array can no longer be invalid here
```suggestion
		log.Info("batch posters array was empty, defaulting to sequencer address")
```